examples/todomvc/config-overrides.js

```diff
@@ -0,0 +1,8 @@
+const {
+  override,
+  addBabelPlugin
+} = require("customize-cra");
+
+module.exports = override(
+  addBabelPlugin('babel-plugin-reusable')
```

Comment:
we can probably delete this now

Reply:
I think we will need a plugin later on for the unit's names, so I'll just leave the skeleton